File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_8.go
package main
import ("fmt"
        
)

func main() {
    var a float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&a)
    var b float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&b)
    var c float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&c)
    var d float64
    fmt.Println ("Vvedite chislo: ")
    fmt.Scan (&d)
    if ((a <= c) && (b <= d)) || ((a <= d) && (b <= c)) {
        fmt.Print ("Da, mojet")
    } else {
        fmt.Print ("Net, ne mojet")
    }
}

File: 2_course/ЯПВУ/Laptev_AV1/Лаптев Александр/LaptevAV-2_8.rb
# encoding utf-8
puts ('Vvedite veshestvennoe chislo: ')
a = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
b = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
c = gets.to_i
puts ('Vvedite veshestvennoe chislo: ')
d = gets.to_i
if ((a <= c) & (b <= d)) || ((a <= d) & (b <= c))
    puts ('Da, mojet')
else
    puts ('Net, ne mojet')
end